easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-v2012-foss-2020a.eb

```
@@ -0,0 +1,42 @@
+name = 'OpenFOAM'
+version = 'v2012'
+
+homepage = 'https://www.openfoam.com/'
+description = """OpenFOAM is a free, open source CFD software package.
+ OpenFOAM has an extensive range of features to solve anything from complex fluid flows
+ involving chemical reactions, turbulence and heat transfer,
+ to solid dynamics and electromagnetics."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = ['https://sourceforge.net/projects/openfoam/files/%(version)s/']
+sources = [SOURCE_TGZ]
+patches = [
+    ('OpenFOAM-%(version)s-cleanup.patch', 0),
```

@furstj suggestion in one easy click;
```suggestion
    ('OpenFOAM-%(version)s-cleanup.patch', 1),
```